server/raft.go

```diff
@@ -11,8 +11,9 @@ import (
 
 	"github.com/hashicorp/raft"
 	"github.com/hashicorp/raft-boltdb"
-	"github.com/liftbridge-io/nats-on-a-log"
-	"github.com/nats-io/go-nats"
+	"github.com/nats-io/nats.go"
+
+	"github.com/liftbridge-io/liftbridge/third/nats-on-a-log"
```

Comment:
Why move this under `/third`?

Reply:
> Why move this under `/third`?

sorry about this. 
it's for my personal test ,  forget to fix.